src/app/loadout/loadout-edit/useEquipDropTargets.tsx

```diff
@@ -0,0 +1,51 @@
+import { DimItem } from 'app/inventory/item-types';
+import { singularBucketHashes } from 'app/loadout-drawer/loadout-utils';
+import { itemCanBeInLoadout } from 'app/utils/item-utils';
+import { DestinyClass } from 'bungie-api-ts/destiny2';
+import { TargetType } from 'dnd-core';
+import { DropTargetHookSpec, useDrop } from 'react-dnd';
+
+/**
+ * Hook to setup dnd drops for loadout editing.
```

Comment:
Would be great to document this - what it is used for, example of how to use it.

Reply:
Can do

Reply:
Done